baybe/serialization/mixin.py

```diff
@@ -29,7 +31,11 @@ def from_dict(cls: Type[_T], dictionary: dict) -> _T:
         Returns:
             The reconstructed object.
         """
-        return converter.structure(dictionary, cls)
+        # In case we use postponed annotations, we must resolve the original type
```

Comment:
the note says `in case` but theres no case distinction here (a la if else or similar), does this also cover the previous version and merely extends it?

Reply:
Yes, that's an extension to handle postponed annotations.